app-text/zathura/zathura-0.5.8.ebuild

```diff
@@ -15,48 +15,48 @@ if [[ ${PV} == *9999 ]]; then
 else
 	SRC_URI="
 		https://github.com/pwmt/zathura/archive/${PV}.tar.gz -> ${P}.tar.gz
-		https://cdn.turret.cyou/354c6d33bfd3bbc67c0047af1328498978eef352/${P}-manpages.tar.xz
 	"
-	KEYWORDS="~amd64 ~arm ~riscv ~x86 ~amd64-linux ~x86-linux"
+	KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86 ~amd64-linux ~x86-linux"
 fi
 
 LICENSE="ZLIB"
-SLOT="0/$(ver_cut 1-2)"
-IUSE="seccomp sqlite synctex test"
+SLOT="0/6.7"
+IUSE="man seccomp synctex test"
 
 RESTRICT="!test? ( test )"
 
-DEPEND="
-	>=dev-libs/girara-0.3.7
-	>=dev-libs/glib-2.50:2
-	sys-apps/file
-	>=sys-devel/gettext-0.19.8
-	x11-libs/cairo
-	>=x11-libs/gtk+-3.22:3
+RDEPEND="
+	dev-libs/json-glib
+	man? ( dev-python/sphinx )
```

Comment:
I personally view this as a [regression in quality](https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0305), especially since the manpage has keybindings listing which is pretty important.

If you can generate the manpage and upload it to some dummy github repo under "release", that would be nice. Otherwise there are also other distros (like debian) which hosts manpages, so you might be able to grab it form there.

Reply:
This change came from an older PR https://github.com/gentoo/gentoo/pull/36137#issuecomment-2041123872